mteb/tasks/Classification/eng/PoemSentimentClassification.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+class PoemSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="PoemSentimentClassification",
+        description="Poem Sentiment is a sentiment dataset of poem verses from Project Gutenberg.",
+        reference="https://arxiv.org/abs/2011.02686",
+        dataset={
+            "path": "poem_sentiment",
+            "revision": "329d529d875a00c47ec71954a1a96ae167584770",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2020-01-01", "2021-01-01"),
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="CC-BY-4.0",
+        socioeconomic_status="mixed",
```

Comment:
I don't know much about the corpus at hand, but at least in some countries in Europe I know that poetry was very much a thing of the elites/nobility. Are we sure that `mixed` is the appropriate label here?

Reply:
Yeah, I agree, and that is what I thought at the beginning. But this is a diverse collection from different countries that could belong to various demographies.